api/comments_api.py

```diff
@@ -19,19 +19,31 @@
 from framework import basehandlers
 from framework import permissions
 from internals import approval_defs
-from internals.review_models import Activity, Approval, Gate, Vote
+from internals.review_models import Activity, Amendment, Approval, Gate, Vote
 from internals import notifier
 
 
-def comment_to_json_dict(comment: Activity) -> dict[str, Any]:
+def amendment_to_json_dict(amendment: Amendment) -> dict[str, Any]:
+  return {
+      'field_name': amendment.field_name,
+      'old_value': amendment.old_value.strip('[]'),
+      'new_value': amendment.new_value.strip('[]'),
+      }
+
+
+def activity_to_json_dict(comment: Activity) -> dict[str, Any]:
+  amendments_json = [
+      amendment_to_json_dict(amnd) for amnd in comment.amendments
+      if amnd.old_value != 'None' or amnd.new_value != '[]']
```

Comment:
why is this 'None'?

Reply:
It is actually the string "None" in the Amendment.old_value field.   When a user makes a change to a feature entry, we compute the Amendment object by converting old and new FeatureEntry field values to strings.  That conversion creates some "None" and "[]" Amendment values.   In the API, I choose to omit the Amendments that are "None" -> "[]" because those are not useful to users.  

I think that those get created simply because there are two ways of representing a missing value for repeated fields. We should actually stop creating them.  I filed issue #2424.